app/services/stories_api/v3/helpers.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module StoriesApi
+  module V3
+    module Helpers
+      def current_user(params)
+        @current_user ||= SupplejackApi::User.find_by_api_key(params[:api_key])
+      end
+
+      def create_exception(error = nil, options = {})
```

Comment:
These Errors don't look to me like Exceptions in ruby (which would respond to raise and rescue). This isn't immediately obvious in the code, and this method name adds to the confusion. `create_error` maybe?